babel/messages/frontend.py

```diff
@@ -118,6 +118,12 @@ def finalize_options(self):
                                        'or the base directory')
 
     def run(self):
+        domains = self.domain.split()
+
+        for domain in domains:
+            self.__run_domain(domain)
+
+    def __run_domain(self, domain):
```

Comment:
Sure, but that's also very nonintuitive behavior.

[Even PEP 8 advises against using the mangly names unless you have a good reason.](https://www.python.org/dev/peps/pep-0008/#id42)

As for the privateness viewpoint, Python's attitude generally is "we're all consenting adults here", i.e. no need to go to lengths to hide the member. The single underscore is already a signal that this is some sort of private API, and you should know what you're doing if you're calling or overriding this.